### PR TITLE
DICT-448 leeuwenhoek follow_up updates

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-448

### CODE REVIEW

File: src/gdcdictionary/schemas/follow_up.yaml [hunk 1]
```diff
@@ -2953,11 +2988,26 @@ properties:
...
+      - Not Reported
+
+  treatment_emergent_adverse_event:
+    $ref:
+      - "_terms.yaml#/treatment_emergent_adverse_event/common"
+    enum:
+      - Definitely
+      - Possibly
+      - Probably
+      - Unlikely
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/follow_up.yaml:3008]: I don't see a difference between Possibly and Unlikely. The seems like 'half-full' and 'half-empty'. Are there metrics related to these terms?

File: src/gdcdictionary/schemas/follow_up.yaml
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/follow_up.yaml:1]: ~I was looking for a new `adverse_event`,`first_event`, enum~
first_event has an additional 'Unknown' not documented in the associated ticket.

**Review by stilesj-uchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/follow_up.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/follow_up.yaml:3008]: I'll defer to @tlicht3 , IMS gets these values from NCI studies and these might be 1:1 to what needs to be populated from the CTSP-KIRC study.

File: src/gdcdictionary/schemas/follow_up.yaml
Comment (bullenc) [reply src/gdcdictionary/schemas/follow_up.yaml:1]: There was an additional row for `first_event` in the table that had `Unknown`, consolidated the two rows

File: src/gdcdictionary/schemas/follow_up.yaml (hunk 1 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/follow_up.yaml:3008]: My guess is that these are the selectable options on some doctors' form.  Based on the CTSP-KIRC trial information, Definitely, Probably, and Possibly fell into one category, and the Unlikely and Unrelated fell into another.

File: src/gdcdictionary/schemas/follow_up.yaml [hunk 2]
```diff
@@ -2953,11 +2988,26 @@ properties:
...
+      - Unrelated
+      - Unknown
       - Not Reported
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/follow_up.yaml:3011]: confused about why this Not Reported was already here. I think it was already there previously for timepoint category and git is just confused?
^^ if so, should we remove "Not Reported" from timepoint_category in the jira?

**Review by wteouchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/follow_up.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/follow_up.yaml:3011]: Yeah I saw that too, I think its just the way that git rendered the diff for changes. I think the timepoint_category enum for Not Reported should remain so we are consistent that all enum type props have `Unknown` and `Not Reported`

Comment (wteouchicago) [reply src/gdcdictionary/schemas/follow_up.yaml:3011]: oh, i mean if timepoint_category already had "Not Reported", then we don't need to claim we added it for Leeuvenhoek in the jira ticket. But not wedded to this in any way.

Comment (bullenc) [reply src/gdcdictionary/schemas/follow_up.yaml:3011]: Ok yes, I see what you mean now. timepoint_category already had `Not Reported` as an enum in the dictionary, removed it as a proposed addition from the ticket.

**Pull request merged**